### PR TITLE
[docs] Update README

### PR DESCRIPTION
**Pull request by stevhliu**
- Updates README to include more recent models for tasks
- Rearranges order of the sections to be more logical (installation -> quickstart)
- Hides some examples in dropdown menus to keep the doc clean
- Removes the [Learn more](https://github.com/huggingface/transformers#learn-more) section since the docs will be refactored soon

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,275 +55,250 @@ limitations under the License.
 </h4>
 
 <h3 align="center">
-    <p>State-of-the-art Machine Learning for JAX, PyTorch and TensorFlow</p>
+    <p>State-of-the-art pretrained models for inference and training</p>
```
Comment (stevhliu) [inline README.md:58]: Removed mention of TF/JAX since it seems like we're kinda moving away from it. Let me know if you want me to keep it!

File: README.md [hunk 2]
```diff
@@ -54,275 +55,250 @@ limitations under the License.
...
 
-```
-git clone https://github.com/huggingface/transformers.git
-cd transformers
-pip install .
-```
+If you own or use a project that you believe should be part of the list, please open a PR to add it!
 
-### With conda
+## Example models
```
Comment (stevhliu) [inline README.md:245]: Let me know if you think we're missing a task or we should highlight a different model!

File: README.md [hunk 3]
```diff
@@ -54,275 +55,250 @@ limitations under the License.
...
-    <p>State-of-the-art Machine Learning for JAX, PyTorch and TensorFlow</p>
+    <p>State-of-the-art pretrained models for inference and training</p>
 </h3>
 
 <h3 align="center">
     <a href="https://hf.co/course"><img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/course_banner.png"></a>
 </h3>
 
-🤗 Transformers provides thousands of pretrained models to perform tasks on different modalities such as text, vision, and audio.
+Transformers is a library of pretrained natural language processing, computer vision, audio, video, and multimodal models for inference and training. Use Transformers to fine-tune models on your data, build inference applications, and for generative AI use cases across multiple modalities.
```
Comment (LysandreJik) [inline README.md:65]:
```suggestion
Transformers is a library of pretrained text, computer vision, audio, video, and multimodal models for inference and training. Use Transformers to fine-tune models on your data, build inference applications, and for generative AI use cases across multiple modalities.
```

File: README.md [hunk 4]
```diff
@@ -54,275 +55,250 @@ limitations under the License.
...
 
-These models can be applied on:
+All 1M+ [model checkpoints](https://huggingface.co/models) from the [Hugging Face Hub](https://huggingface.com/models) are supported by Transformers.
```
Comment (LysandreJik) [inline README.md:67]:
```suggestion
All 500k+ [model checkpoints tagged as `transormers`](https://huggingface.co/models?library=transformers&sort=trending) from the [Hugging Face Hub](https://huggingface.com/models) are supported by Transformers.
```

File: README.md [hunk 5]
```diff
@@ -54,275 +55,250 @@ limitations under the License.
...
-* 🖼️ Images, for tasks like image classification, object detection, and segmentation.
-* 🗣️ Audio, for tasks like speech recognition and audio classification.
+Explore the [Hub](https://huggingface.com/) today to find a model and use Transformers to help you get started right away.
 
-Transformer models can also perform tasks on **several modalities combined**, such as table question answering, optical character recognition, information extraction from scanned documents, video classification, and visual question answering.
+## Installation
 
-🤗 Transformers provides APIs to quickly download and use those pretrained models on a given text, fine-tune them on your own datasets and then share them with the community on our [model hub](https://huggingface.co/models). At the same time, each python module defining an architecture is fully standalone and can be modified to enable quick research experiments.
+Transformers works with Python 3.6+ [PyTorch](https://pytorch.org/get-started/locally/) 1.1.0+, [TensorFlow](https://www.tensorflow.org/install/pip) 2.0+, and [Flax](https://flax.readthedocs.io/en/latest/).
```
Comment (LysandreJik) [inline README.md:73]: cc @ydshieh on a quick review re. latest supported versions (I think it's Python 3.8 and torch 2.0)

File: README.md [hunk 6]
```diff
@@ -54,275 +55,250 @@ limitations under the License.
...
+Instantiate a pipeline and specify model to use for text generation. The model is downloaded and cached so you can easily reuse it again. Finally, pass some text to prompt the model.
 
-In order to celebrate the 100,000 stars of transformers, we have decided to put the spotlight on the
-community, and we have created the [awesome-transformers](./awesome-transformers.md) page which lists 100
-incredible projects built in the vicinity of transformers.
+```py
+from transformers import pipeline
 
-If you own or use a project that you believe should be part of the list, please open a PR to add it!
+pipeline = pipeline(task="text-generation", model="Qwen/Qwen2.5-1.5B")
```
Comment (LysandreJik) [inline README.md:114]: Good choice of model!

File: README.md [hunk 7]
```diff
@@ -54,275 +55,250 @@ limitations under the License.
...
+<details>
+<summary>Audio</summary>
 
-Follow the installation pages of Flax, PyTorch or TensorFlow to see how to install them with conda.
+- Audio classification with [Whisper](https://huggingface.co/openai/whisper-large-v3-turbo)
+- Automatic speech recognition with [Moonshine](https://huggingface.co/UsefulSensors/moonshine)
+- Keyword spotting with [Wav2Vec2](https://huggingface.co/superb/wav2vec2-base-superb-ks)
+- Speech to speech generation with [Moshi](https://huggingface.co/kyutai/moshiko-pytorch-bf16)
+- Text to audio with [MusicGen](https://huggingface.co/facebook/musicgen-large)
+- Text to speech with [Bark](https://huggingface.co/suno/bark)
```
Comment (LysandreJik) [inline README.md:259]: cc @eustlb @Vaibhavs10 in case you see better/others that would fit here

File: README.md [hunk 8]
```diff
@@ -54,275 +55,250 @@ limitations under the License.
...
+<summary>Computer vision</summary>
 
-**[All the model checkpoints](https://huggingface.co/models)** provided by 🤗 Transformers are seamlessly integrated from the huggingface.co [model hub](https://huggingface.co/models), where they are uploaded directly by [users](https://huggingface.co/users) and [organizations](https://huggingface.co/organizations).
+- Automatic mask generation with [SAM](https://huggingface.co/facebook/sam-vit-base)
+- Depth estimation with [DepthPro](https://huggingface.co/apple/DepthPro-hf)
+- Image classification with [DINO v2](https://huggingface.co/facebook/dinov2-base)
+- Keypoint detection with [SuperGlue](https://huggingface.co/magic-leap-community/superglue_outdoor)
+- Object detection with [RT-DETRv2](https://huggingface.co/PekingU/rtdetr_v2_r50vd)
+- Universal segmentation with [OneFormer](https://huggingface.co/shi-labs/oneformer_ade20k_swin_large)
+- Video classification with [VideoMAE](https://huggingface.co/MCG-NJU/videomae-large)
```
Comment (LysandreJik) [inline README.md:272]: cc @qubvel @merveenoyan in case you see better/others that would fit here

File: README.md [hunk 9]
```diff
@@ -54,275 +55,250 @@ limitations under the License.
...
 
-To check if each model has an implementation in Flax, PyTorch or TensorFlow, or has an associated tokenizer backed by the 🤗 Tokenizers library, refer to [this table](https://huggingface.co/docs/transformers/index#supported-frameworks).
+- Audio or text to text with [Qwen2-Audio](https://huggingface.co/Qwen/Qwen2-Audio-7B)
+- Document question answering with [LayoutLMv3](https://huggingface.co/microsoft/layoutlmv3-base)
+- Image or text to text with [Qwen-VL](https://huggingface.co/Qwen/Qwen2.5-VL-3B-Instruct)
+- Image captioning with [Llava](https://huggingface.co/llava-hf/llava-1.5-7b-hf)
+- Table question answering with [TAPAS](https://huggingface.co/google/tapas-base)
+- Unified multimodal understanding and generation with [Emu3](https://huggingface.co/BAAI/Emu3-Gen)
+- Vision to text with [Llava-OneVision](https://huggingface.co/llava-hf/llava-onevision-qwen2-0.5b-ov-hf)
+- Visual question answering with [BLIP-2](https://huggingface.co/Salesforce/blip2-opt-2.7b)
```
Comment (LysandreJik) [inline README.md:286]: cc @zucchini-nlp in case you see better/others that would fit here

File: README.md [hunk 10]
```diff
@@ -54,275 +55,250 @@ limitations under the License.
...
+<summary>NLP</summary>
 
-## Learn more
+- Masked word completion with [ModernBERT](https://huggingface.co/answerdotai/ModernBERT-base)
+- Named entity recognition with [Gemma](https://huggingface.co/google/gemma-2-2b)
+- Question answering with [Mixtral](https://huggingface.co/mistralai/Mixtral-8x7B-v0.1)
+- Summarization with [BART](https://huggingface.co/facebook/bart-large-cnn)
+- Translation with [T5](https://huggingface.co/google-t5/t5-base)
+- Text generation with [Llama](https://huggingface.co/meta-llama/Llama-3.2-1B)
+- Text classification with [Qwen](https://huggingface.co/Qwen/Qwen2.5-0.5B)
```
Comment (LysandreJik) [inline README.md:299]: cc @ArthurZucker in case you see better/others that would fit here

**Review by LysandreJik — APPROVED** [review]
Thanks @stevhliu! Left a few comments and pinged relevant team members to quickly verify the models that were shared. Thanks for the refactor!

**Review by shethaadit — APPROVED** [review]

File: README.md (hunk 8 above)
Comment (qubvel) [reply README.md:272]:
```
 - Pose Estimation with [VitPose](https://huggingface.co/usyd-community/vitpose-base-simple)
 - Keypoint matching with [SuperGlue](https://huggingface.co/magic-leap-community/superglue)
```

File: README.md (hunk 9 above)
Comment (zucchini-nlp) [reply README.md:286]: Can change these two, imo BLIP better suits if captioning only task:
Image captioning with BLIP-2
Visual question answering with LLaVA

Plus maybe add these two, feel free to word choice this better:
OCR-based Document Understanding with GOT-OCR2 
Visual Referring Expression Segmentation with Kosmos2

File: README.md (hunk 7 above)
Comment (stevhliu) [reply README.md:259]: gentle ping @eustlb @Vaibhavs10 if there are better audio models to add here :)

**Comment by stevhliu** [comment]
Merging now and we can update later :)
